composer.json

```diff
@@ -19,7 +19,8 @@
         "php": ">=5.3.9",
         "doctrine/common": "~2.3",
         "twig/twig": "~1.18",
-        "psr/log": "~1.0"
+        "psr/log": "~1.0",
+        "symfony/templating": "~2.8"
```

Comment:
How about putting it in the suggest section then ? And as @fabpot said, I think the dependency in the framework bundle is enough (having to add one line to the composer.json file is fine I think, with a proper upgrade message ofc)